packages/sdks/src/components/blocks/blocks-wrapper.lite.tsx

```diff
@@ -61,8 +68,23 @@ export default function BlocksWrapper(props: BlocksWrapperProps) {
     },
   });
 
+  onMount(() => {
+    useTarget({
+      reactNative: () => {
+        if (isEditing()) {
+          props.path &&
+            blocksWrapperRef.setAttribute('builder-path', props.path);
+          props.parent &&
+            blocksWrapperRef.setAttribute('builder-parent-id', props.parent);
+        }
```

Comment:
a comment on this would be good (react native strips off custom HTML attributes, so we have to manually set them here)

Reply:
thanks! added